scripts/svm/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const CIRCLE_IRIS_API_URL_DEVNET = "https://iris-api-sandbox.circle.com";
+export const CIRCLE_IRIS_API_URL_MAINNET = "https://iris-api.circle.com";
+export const HUB_POOL_ADDRESS_MAINNET = "0x75Baa2565C710644679F56E21ce3FF54Ef2b6192";
+export const HUB_POOL_ADDRESS_SEPOLIA = "0x75Baa2565C710644679F56E21ce3FF54Ef2b6192";
+export const SOLANA_USDC_MAINNET = "EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v";
+export const SOLANA_USDC_DEVNET = "4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU";
```

Comment:
should we hardcode them? we might have new deployments, so maybe better to set these in env?

Reply:
I've moved the `HUB_POOL_ADDRESS` to an env var but kept `SOLANA_USDC_DEVNET` and `SOLANA_USDC_MAINNET` as constants as these are managed by CCTP